example-scripts/01-sample-script.py

```diff
@@ -0,0 +1,57 @@
+"""Module 1 - 
+
+* What challenges do you see in this code? 
+* How could we make it more concise? 
+* How could we make it less - copy pasta and more reusable? 
+* What would happen if you needed to process multiple csv files? 
+* Is this code easily shareable?
+"""
+
+import pandas as pd
+import os
+
+print(os.getcwd())
```

Comment:
By using temperature and snowfall we have units that we could potentially convert. there are conversion tools but we could also create small functions to implement the conversions.